includes/misc-functions.php

```diff
@@ -580,9 +579,9 @@ function give_get_newsletter() {
 
 		<form action="//givewp.us3.list-manage.com/subscribe/post?u=3ccb75d68bda4381e2f45794c&amp;id=12a081aa13"
 		      method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate"
-		      target="_blank" novalidate>
+		      target="_blank">
 			<div class="give-newsletter-confirmation">
-				<p><?php esc_html_e( 'Thanks for Subscribing!', 'give' ); ?> :)</p>
+				<p><?php esc_html_e( 'A confirmation mail is being sent to your email ID. Kindly conform your subscription over there! :)', 'give' ); ?></p>
```

Comment:
@DevinWalker Need you help to update this text

Reply:
@raftaar1191 Please use the following text:
> To complete your subscription, click the confirmation link in your email. Thank you!